assignments/assignment04.rb

```diff
@@ -0,0 +1,174 @@
+# ========================================================================================
+# Assignment 4
+# ========================================================================================
+
+# ========================================================================================
+#  Problem 1 - Fibonacci
+
+def fib(n)
+  if (0..1).include? n
```

Comment:
you have a test for the base case, but you don't do anything with this. In fact, the way you've written this, the ruby interpreter will read this as:
```ruby
if (0..1).include? n
  (fib(n-1) + fib(n-2))
end
```
and the rest of the code will be part of your fib method. I don't think this is what you intended.

It also tells me that you never actually tried to execute this code.